src/unix/linux.c

```diff
@@ -2308,46 +2308,70 @@ static int uv__get_cgroupv2_constrained_cpu(const char* cgroup,
                                             uv__cpu_constraint* constraint) {
   char path[256];
   char buf[1024];
-  unsigned int weight;
   int cgroup_size;
+  long long min_quota;
+  long long quota;
+  long long limit;
+  long long period;
+  char* end;
+  const char* cgroup_mount;
```

Comment:
```suggestion
  static const char cgroup_mount[] = "/sys/fs/cgroup";
```
Since it never changes (and please move it to the top.)

Reply:
Done